apps/application-system/api/src/app/modules/application/e2e/application.spec.ts

```diff
@@ -56,6 +57,17 @@ class MockContentfulRepository {
   }
 }
 
+const mockConfig = {
+  jwtSecret: 'supersecret',
+  clientLocationOrigin: 'http://localhost:4200',
```

Comment:
If this is for the application-system frontend, then the port should be `4242` no?